Feature/investment screener (#52)
highcharts/connectors-morningstar: docs/connectors/morningstar/screeners/investment-screener.md

> Investment Screener connector

Reviewer: Since these are used as Docs titles on Docs sidebar, I would remove the connector bit
```suggestion
Investment Screener
```

=============================

> Using Morningstar Investment Screener endpoint you are able filter Morningstar's database of global investments using hundreds of data points including Morningstar prorietary data.

Reviewer: ```suggestion
Using Morningstar Investment Screener endpoint allows you to filter Morningstar's database of global investments using hundreds of data points including Morningstar proprietary data.
```

How to use Investment Screener
----------------

Data can be filtered on any data point which is comprehensively covered by Morningstar, including proprietary data such as:

 * Morningstar Sustainability Ratings
 * Analyst Rating
 * Fair Value
 * Style Box
 * With the solution, you can develop a sophisticated screening tool offering dozens of filters for advanced users.

You can also use it to power predefined screeners for common investment research interests such as Top Fixed Income Funds, Top Rated US Bond Index Funds, Sustainable Investments, and so on.

For more details, see [Morningstar's Investment Screener API].

<!-- Links -->
[Morningstar's Investment Screener API]: https://developer.morningstar.com/direct-web-services/documentation/api-reference/screener/investment-screener

> this connector is designed to be interacted with using some external buttons, that might filter data on the backend, provide pagiantion as well as sorting.

Reviewer: ```suggestion
this connector is designed to be interacted with using external buttons, that might filter data on the backend, provide pagination as well as sorting.
```

Here is an example of how to use the Investment Screener connector:

```js
const screenerConnector = MC.InvestmentScreenerConnector({
    page: 1,
    pageSize: 20,

>     langageId: 'en-GB',

Reviewer: ```suggestion
    languageId: 'en-GB',
```
